script.service.janitor/resources/skins/Default/720p/DialogLogViewer.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
I'm not totally sure what you mean by this. Should I rename the files themselves or do I have to state such a name inside the XML structure somewhere?